test/storage/tile_compression_select_test.cpp

```diff
@@ -0,0 +1,80 @@
+#include <memory>
+#include <math.h>
+#include "sql/testing_sql_util.h"
+#include "catalog/catalog.h"
+#include "catalog/database_catalog.h"
+#include "storage/database.h"
+#include "storage/data_table.h"
+#include "concurrency/testing_transaction_util.h"
+#include "common/harness.h"
+#include "executor/create_executor.h"
+#include "planner/create_plan.h"
+#include "storage/tile_group_iterator.h"
+#include "storage/tile_group.h"
+#include "storage/dictionary_encoding_tile.h"
+
+namespace peloton {
+namespace test {
+class CompressionSelectTest : public PelotonTest {};
```

Comment:
Rename to TileCompressionSQLTests and rename file to tile_compression_sql_test.cpp to comply with other SQL testing conventions.